lua/cfc_time/server/time/sv_time_tracking.lua

```diff
@@ -0,0 +1,81 @@
+CFCTime.ctime = {}
+local ctime = CFCTime.ctime
+
+local function now()
+    return os.time()
+end
+
+-- <steamID64> = { joined = <timestamp>, departed = <timestamp> | nil, initialTime = <float> }
+ctime.pendingUpdates = {}
+ctime.updateTimerName = "CFC_Time_UpdateTimer"
+ctime.lastUpdate = now()
+
+function ctime:updateTimes()
```

Comment:
why make these functions, methods, why not ctime.updateTimes
there is only ever one instance of the ctime table

Reply:
`self` nice